examples/camera-controller/webrtc_manager/WebRTCManager.cpp

```diff
@@ -70,19 +70,46 @@ void WebRTCManager::Init()
     // TODO:: mWebRTCRequestorServer.Init();
 }
 
+CHIP_ERROR WebRTCManager::SetRemoteDescription(uint16_t webRTCSessionID, const std::string & sdp)
+{
+    if (!mPeerConnection)
+    {
+        ChipLogError(NotSpecified, "Cannot set remote description: mPeerConnection is null");
+        return CHIP_ERROR_INCORRECT_STATE;
+    }
+
+    ChipLogProgress(NotSpecified, "WebRTCManager::SetRemoteDescription");
+    mPeerConnection->setRemoteDescription(sdp);
+
+    // Schedule the ProvideICECandidates() call to run asynchronously.
+    DeviceLayer::SystemLayer().ScheduleLambda([this, webRTCSessionID]() { ProvideICECandidates(webRTCSessionID); });
+
+    return CHIP_NO_ERROR;
+}
+
 CHIP_ERROR WebRTCManager::Connnect(Controller::DeviceCommissioner & commissioner, NodeId nodeId, EndpointId endpointId)
 {
     ChipLogProgress(NotSpecified, "Attempting to establish WebRTC connection to node 0x" ChipLogFormatX64 " on endpoint 0x%x",
                     ChipLogValueX64(nodeId), endpointId);
 
-    mWebRTCProviderClient.Init(commissioner, nodeId, endpointId);
+    FabricIndex fabricIndex       = commissioner.GetFabricIndex();
+    const FabricInfo * fabricInfo = commissioner.GetFabricTable()->FindFabricWithIndex(fabricIndex);
+    VerifyOrReturnError(fabricInfo != nullptr, CHIP_ERROR_INCORRECT_STATE);
+
+    uint64_t fabricId = fabricInfo->GetFabricId();
+    ChipLogProgress(NotSpecified, "Commissioner is on Fabric ID 0x" ChipLogFormatX64, ChipLogValueX64(fabricId));
+
+    chip::ScopedNodeId peerId(nodeId, fabricIndex);
+
+    mWebRTCProviderClient.Init(peerId, endpointId);
 
     rtc::InitLogger(rtc::LogLevel::Warning);
 
     // Create the peer connection
     rtc::Configuration config;
     mPeerConnection = std::make_shared<rtc::PeerConnection>(config);
 
+    // Use std::cout to print out to console
```

Comment:
But _why_ instead of doing normal logging?  If people want normal logging on console, they can do that.  If they want it somewhere else, they can do that.  Why are we forcing console here for these specific bits?